accord-core/src/test/java/accord/burn/TopologyUpdate.java

```diff
@@ -0,0 +1,248 @@
+package accord.burn;
+
+import accord.api.KeyRange;
+import accord.api.TestableConfigurationService;
+import accord.local.Command;
+import accord.local.Node;
+import accord.local.Status;
+import accord.topology.KeyRanges;
+import accord.topology.Shard;
+import accord.topology.Topology;
+import accord.txn.Dependencies;
+import accord.txn.Timestamp;
+import accord.txn.Txn;
+import accord.txn.TxnId;
+import accord.utils.MessageTask;
+import com.google.common.base.Preconditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.*;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.stream.Stream;
+
+public class TopologyUpdate
+{
+    private static final Logger logger = LoggerFactory.getLogger(TopologyUpdate.class);
+
+    private static class CountDownFuture<T> extends CompletableFuture<T>
```

Comment:
I think we should consider inserting a cassandra.jar in our dependencies for now, so we can use the same facilities we do in mainline until we can figure out a good way to share code. WDYT?